src/sentry/static/sentry/app/views/groupEventDetails.jsx

```diff
@@ -89,14 +90,22 @@ const GroupEventDetails = React.createClass({
                 {group.status === 'ignored' &&
                   <MutedBox statusDetails={group.statusDetails} />}
                 {group.status === 'resolved' &&
-                  group.statusDetails.inNextRelease &&
+                  group.statusDetails.inRelease &&
                   <div className="box">
                     <span className="icon icon-checkmark" />
                     <p>
-                      {t(
-                        `This issue has been marked as being resolved in the next
-                      release. Until then, you will not get notified about new
-                      occurrences.`
+                      {tct(
+                        `This issue has been marked as being resolved in version
+                         [version].`,
```

Comment:
I would say "has been marked as resolved as of version [version]" since its not really "in version" when you're saying "in the next release"